Stratis.Bitcoin/Configuration/Logging/LoggingConfiguration.cs

```diff
@@ -40,9 +71,6 @@ static LoggingConfiguration()
 
             // Installs handler to be called when NLog's configuration file is changed on disk.
             LogManager.ConfigurationReloaded += NLogConfigurationReloaded;
-
-            // Load the key to name-space mappings.
-            keyCategories = LoadKeyCategories();
```

Comment:
You can see I've deleted quite a few lines after moving it upstairs. So it looks cleaner now.